app/questionnaire/questionnaire_store_updater.py

```diff
@@ -35,7 +35,7 @@ def save_new_list_item_answers(self, form, list_name):
         self.save_answers(form, False)
 
     def _update_questionnaire_store_with_form_data(self, form_data):
-        answer_ids_for_question = self._schema.get_answer_ids_for_question(
+        answer_ids_via_question = self._schema.answer_ids_via_question(
```

Comment:
I'm unsure about the use of `via` in general, but the variable in this case should remain as `answer_ids_for_question` I think.